cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/pipeline/DeleteScheduleStage.java

```diff
@@ -63,10 +65,11 @@ public void process(ApplicationWithPrograms input) throws Exception {
       // delete schedules that existed in the old app spec, but don't anymore
       ScheduleSpecification scheduleSpec = entry.getValue();
       ProgramType programType = ProgramType.valueOfSchedulableType(scheduleSpec.getProgram().getProgramType());
```

Comment:
Couple of concerns:
- I thought we wanted to stop doing this (deleting schedules that were removed from the app spec). Because otherwise, if I add a schedule using REST, and then update the app, the schedule will be deleted. 
- What happens if I had an (old-style) time schedule in my app, now I change my app to have the same, but now it is a program schedule with a time trigger? Will that lead to deletion of the schedule?